src/test/unit_tests/jest_tests/test_nc_account_cli.test.js

```diff
@@ -2218,6 +2251,52 @@ async function exec_manage_cli_add_empty_option(command, option) {
     return res;
 }
 
+/**
+ * exec_manage_cli_add_leading_space_option modifies the command by appending a flag with a value(quoted or non-quoted) that includes a leading space to test CLI parsing behavior
+ * @param {string} command
+ * @param {string} option
+ * @param {string} value
+ * @param {"s" | "qs" | "sq" | "default"} quoted
+ *
+ * @examples
+ * - await exec_manage_cli_add_leading_space_option("cli_command", "flag", "val", quoted_type.SPACE_ONLY);
+ *   // cli_command --flag= val  (Space before value)
+ *
+ * - await exec_manage_cli_add_leading_space_option("cli_command", "flag", "val", quoted_type.QUOTE_SPACE);
+ *   // cli_command --flag=" val"  (quote then space)
+ *
+ * - await exec_manage_cli_add_leading_space_option("cli_command", "flag", "val", quoted_type.SPACE_QUOTE);
+ *   // cli_command --flag= "val"  (space then quote)
+ *
+ * - await exec_manage_cli_add_leading_space_option("cli_command", "flag", "val");
+ *   // cli_command --flag=val  (No special formatting)
+ */
+async function exec_manage_cli_add_leading_space_option(command, option, value, quoted = "default") {
```

Comment:
I'm not sure you need to add this default value in the function argument
```suggestion
async function exec_manage_cli_add_leading_space_option(command, option, value, quoted) {
```

Reply:
We need this to make the `quoted` argument optional. Thanks